lib/fluent/plugin/out_http.rb

```diff
@@ -202,7 +215,7 @@ def create_request(chunk, uri)
         req.basic_auth(@auth.username, @auth.password)
       end
       set_headers(req)
-      req.body = chunk.read
+      req.body = @json_array ? "[#{chunk.read[0...-1]}]" : chunk.read
```

Comment:
Why use `[0...-1]`? `#read` returns String object.

Reply:
I use `[0...-1]` to exclude the last character from the string:
- `"[{<data>},{<data>},]"` is not valid json string
- `"[{<data>},{<data>}]"` is valid json string afaik

Should I use another approach?

Reply:
Ah, I see. If so, please use `chop!` method instead.
`[0...-1]` creates new object and it consumes bigger memory for large string.